src/cow-react/modules/swap/hooks/useHandleSwapCallback.ts

```diff
@@ -0,0 +1,28 @@
+import { PriceImpact } from 'hooks/usePriceImpact'
+import { useSwapFlowContext } from './useSwapFlowContext'
+import { useEthFlowContext } from './useEthFlowContext'
+import { useSwapActionHandlers } from 'state/swap/hooks'
+import { useCallback } from 'react'
+import { logTradeFlow } from '@cow/modules/trade/utils/logger'
+import { swapFlow } from '@cow/modules/swap/services/swapFlow'
+import { ethFlow } from '@cow/modules/swap/services/ethFlow'
+
+export function useHandleSwapCallback(priceImpactParams: PriceImpact): () => Promise<void> {
```

Comment:
do we really need to include `Callback` in the name. This hook will return a function called `handleSwap`, not sure what the callback adds